content/sigs/advocacy-and-outreach/outreach-programs/index.adoc

```diff
@@ -48,9 +52,11 @@ The https://developers.google.com/season-of-docs/[Google Season of Doc]
 program brings together open source and technical writers communities for the benefit of both.
 The program raises awareness of open source, of docs, and of technical writing.
 
-To contact us regarding this program,
-please visit the
-link:/sigs/docs[documentation SIG] page.
+The Jenkins organization would like to participate in this program.
+
+Please see the
+link:/sigs/docs[documentation SIG] page
+for contact information, and let us know if you'd like to help us participate in this program.
```

Comment:
Could we leave GSoD here if we invite the community to help us join the program?

Reply:
If you feel strongly that it should remain, then yes, I have no strong objection to leave GSoD here.

I thought that our successful experience with GSoC and my botched attempt to submit to GSoD taught us that  strong existing leaders in the community (you, Oleg, etc.) are the critical creators and coordinators of submissions to an outreach program.  The outreach program and the submission to an outreach program seems first to require a very strong push from leaders inside the community, then contributors from outside the community join and assist.

I'm not sure that mentioning GSoD as a future plan here will increase involvement in GSoD.

Reply:
The reason I have GSoD on this page is because we have a QR code on [this poster}(https://github.com/martinda/jenkins-gsoc-gsod-outreachy-poster/blob/master/poster.svg) that links to this page, and the poster mentions GSoD. So I am not too sure what to do now. The poster is to promote those community programs. I thought I'd put the same programs on this page as on the poster. Maybe I need a different section for GSoD?

Reply:
No objection from me to keep it in the page.